test/services/cli-http-client.test.js

```diff
@@ -65,7 +71,6 @@ describe('services', () => {
       })
       .it('correctly serializes array parameters', async () => {
         const client = new CliRequestClient('bleh', logger);
-        expect(client.keytarWord).to.equal('');
```

Comment:
Couldn't get why we're removing this from here?

Reply:
I had added it, moved it to the test case above where I am setting false for keytar, that's all